docs/man_pages/project/configuration/generate.md

```diff
@@ -0,0 +1,33 @@
+generate
+==========
+
+Usage | Synopsis
+------|-------
+General | `$ appbuilder generate <Command>`
+
+Generates images from a single high-resolution one and saves them to the current project
```

Comment:
Generates image resources for all platforms from a single high-resolution image and saves them to the `App_Resources` folder of the current project.